includes/class-apple-news.php

```diff
@@ -93,6 +93,22 @@ class Apple_News {
 	 */
 	public static $maturity_ratings = array( 'KIDS', 'MATURE', 'GENERAL' );
 
+	/**
+	 * Maps a capability to a specific post type, with support for
+	 * custom post types.
+	 *
+	 * @param string $capability The capability to map.
+	 * @param string $post_type  The post type to map against.
+	 * @return string The mapped capability.
+	 */
+	public static function get_capability_for_post_type( $capability, $post_type ) {
+		$post_type_object = get_post_type_object( $post_type );
+
+		return ! empty( $post_type_object->cap->{$capability} )
+			? $post_type_object->cap->{$capability}
+			: '';
```

Comment:
The fallback could be `do_not_allow`, which is how Core explicitly blocks a cap.

Reply:
Good idea.